semgrep/tests/e2e/snapshots/test_rule_parser/test_rule_parser__failure__error_messages/badpattern/error-in-color.txt

```diff
@@ -2,5 +2,5 @@ Running 1 rules...
 [31m[22m[24m[31m[41m[22m[24m[[0m[38;5;231m[41m[1m[24mERROR[0m[31m[41m[22m[24m][0m Semgrep Core — Pattern parse error
 An error occurred while invoking the Semgrep engine. Please help us fix this by creating an issue at https://github.com/returntocorp/semgrep
 
-In rule eqeq-is-bad: Invalid pattern for Python
+In rule rules.syntax.eqeq-is-bad: Invalid pattern for Python
```

Comment:
@brendongo Here is an illustrative change

Reply:
I'm a bit concerned because full paths can be quite long

Reply:
Seems fine to have more info. if anything, easier to find where the offending rule is now and rules from semgrep.dev shouldnt get any change in behavior